mailbox/api/src/main/java/org/apache/james/mailbox/events/MailboxListener.java

```diff
@@ -63,6 +63,18 @@ default ExecutionMode getExecutionMode() {
         return ExecutionMode.SYNCHRONOUS;
     }
 
+
+    /**
+     * Indicate if the event is taken into account by the listener.
+     * Is used internally by the event(Event event) method.
+     *
+     * @param event not null
+     * @return isUsed
+     */
+    default boolean isUsed(Event event) {
```

Comment:
what do you think of isHandling instead of isUsed?
btw the param and return javadoc seem useless here

Reply:
I agree isHandling seems better.